spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -631,11 +631,255 @@
         [
           404,
           {}, # response headers
-          {}  # response body
+          {} # response body
         ]
       end
       expect { api_client.remove_testers("bad_project_number", emails) }
         .to raise_error(ErrorMessage::INVALID_PROJECT)
     end
   end
+
+  describe '#add_tester_group' do
+    let(:headers) do
+      {
+        'Authorization' => 'Bearer auth_token',
+        'Content-Type' => 'application/json',
+        'X-Client-Version' => "fastlane/#{Fastlane::FirebaseAppDistribution::VERSION}"
+      }
+    end
+    let(:project_number) { "project_number" }
+    let(:group_alias) { "group_alias" }
+    let(:display_name) { "Display name" }
+
+    it 'is successful' do
+      payload = {
+        name: "projects/#{project_number}/groups/#{group_alias}",
+        displayName: display_name
+      }
+      stubs.post("/v1/projects/#{project_number}/groups?groupId=#{group_alias}", payload.to_json, headers) do |env|
+        [
+          200,
+          {}, # response headers
+          {} # response body
+        ]
+      end
+
+      result = api_client.create_group(project_number, group_alias, display_name)
+
+      expect(result.success?).to eq(true)
```

Comment:
Consider returning the body of the response i.e. the `group` hash, and verifying that in the test.